admin/spec/back-end/service/prepare-check.service.spec.js

```diff
@@ -50,15 +51,20 @@ describe('prepare-check.service', () => {
     }
   })
 
-  it('should add each check with the expected cache key', async () => {
-    let actualKey
+  it('should add each check with the expected cache key and a lookup item', async () => {
+    let actualPreparedCheckKey, actualLookupKey
     check.schoolPin = 'schoolPin'
     check.pupilPin = 'pupilPin'
+    const expectedLookupKey = `check-started-check-lookup:${check.checkCode}`
+    const expectedPreparedCheckKey = `preparedCheck:${check.schoolPin}:${check.pupilPin}`
     spyOn(redisService, 'setMany').and.callFake((batch, ttl) => {
-      actualKey = batch[0].key
+      console.dir(batch)
```

Comment:
do you need that extra noise in your test?

Reply:
Please delete the console.dir